dashboard/lib/services/lti.rb

```diff
@@ -27,6 +28,13 @@ def self.initialize_lti_user(id_token)
     user
   end
 
+  def self.create_lti_user_identity(user)
+    auth_option = user.authentication_options.find {|ao| ao.credential_type == AuthenticationOption::LTI_V1}
```

Comment:
Here can we use the `lti?` method from the auth options model like `user.authentication_options.find(&:lti?)`?